src/asm/rgbasm.1

```diff
@@ -22,74 +22,170 @@
 The
 .Nm
-program creates an object file from an assembly source file.
+program creates an RGB object file from an assembly source file.
 The input
 .Ar file
 can be a file path, or
 .Cm \-
 denoting
 .Cm stdin .
-Its arguments are as follows:
+.Pp
+Note that options can be abbreviated as long as the abbreviation is unambiguous:
+.Fl Fl verb
+is
+.Fl Fl verbose ,
+but
+.Fl Fl ver
+could also be
+.Fl Fl version .
+The arguments are as follows:
 .Bl -tag -width Ds
-.It Fl b Ar chars
+.It Fl b , Fl Fl binary-digits Ar chars
 Change the two characters used for binary constants.
 The defaults are 01.
-.It Fl D Ar name Ns Op = Ns Ar value
-Add string symbol to the compiled source code. This is equivalent to
-.Ar name
-.Cm EQUS
-.Qq Ar "value"
-in code. If a value is not specified, a value of 1 is given.
-.It Fl E
+.It Fl D , Fl Fl define Ar name Ns Oo = Ns Ar value Oc
+Add a string symbol to the compiled source code.
+This is equivalent to
+.Ql Ar name Ic EQUS Qq Ar value
+in code, or
+.Ql Ar name Ic EQUS Qq 1
+if
+.Ar value
+is not specified.
+.It Fl E , Fl Fl export-all
 Export all labels, including unreferenced and local labels.
-.It Fl g Ar chars
-Change the four characters used for binary constants.
+.It Fl g , Fl Fl gfx-chars Ar chars
+Change the four characters used for gfx constants.
 The defaults are 0123.
-.It Fl h
+.It Fl h , Fl Fl halt-without-nop
 By default,
 .Nm
 inserts a
-.Sq nop
+.Ic nop
 instruction immediately after any
-.Sq halt
+.Ic halt
 instruction.
 The
 .Fl h
 option disables this behavior.
-.It Fl i Ar path
+.It Fl i , Fl Fl include Ar path
 Add an include path.
-.It Fl L
+.It Fl L , Fl Fl preserve-ld
 Disable the optimization that turns loads of the form
-.Sy LD [$FF00+n8],A
+.Ic LD [$FF00+n8],A
 into the opcode
-.Sy LDH [$FF00+n8],A
+.Ic LDH [$FF00+n8],A
 in order to have full control of the result in the final ROM.
-.It Fl M Ar dependfile
+.It Fl M , Fl Fl dependfile Ar dependfile
 Print
 .Xr make 1
 dependencies to
 .Ar dependfile .
-.It Fl o Ar outfile
+.It Fl o , Fl Fl output Ar outfile
 Write an object file to the given filename.
-.It Fl p Ar pad_value
+.It Fl p , Fl Fl pad-value Ar pad_value
 When padding an image, pad with this value.
 The default is 0x00.
-.It Fl r Ar recursion_depth
+.It Fl r , Fl Fl recursion-depth Ar recursion_depth
 Specifies the recursion depth at which RGBASM will assume being in an infinite loop.
-.It Fl V
+.It Fl V , Fl Fl version
 Print the version of the program and exit.
-.It Fl v
+.It Fl v , Fl Fl verbose
 Be verbose.
+.It Fl W , Fl Fl warning Ar warning
+Set warning flag
+.Ar warning .
+A warning message will be printed if
+.Ar warning
+is an unknown warning flag.
+See the
+.Sx DIAGNOSTICS
+section for a list of warnings.
 .It Fl w
-Disable warning output.
+Disable all warning output, even when turned into errors.
+.El
+.Sh DIAGNOSTICS
+Warnings are diagnostic messages that indicate possibly erroneous behavior that does not necessarily compromise the assembling process.
```

Comment:
I wasn’t saying to put every sentence on a whole line, only that sentences can’t start in the middle of a line. You can still wrap the sentence itself to 80 characters.

    .\" bad
    The quick brown fox jumps over the lazy dog. Sphinx of
    black quartz, judge my vow.

    .\" fine
    The quick brown fox jumps over the lazy dog.
    Sphinx of black quartz, judge my vow.

    .\" also fine
    The quick brown fox jumps over the lazy
    dog.
    Sphinx of black quartz,
    judge my vow.

Reply:
Eh, whatever, it's done now. Besides, checkpatch didn't complain, somehow.